### PR TITLE
Improve validation for unintialized ensembles in manual update and evaluate ensemble

### LINKED ISSUE #8691: No validation for uninitialized ensembles in manual update/evaluate experiment
### What happened? (You can include a screenshot if it helps explain)

Open ert gui and create a new experiment / ensemble in the manage experiments window, then attempt  to run either manual update or evaluate ensemble. 

image

### What did you expect to happen?

Should validate and disable run experiment button.

### steps to reproduce

```Python
ert gui poly.ert
Create new experiment / ensemble
Run either manual update or evaluate ensemble
```

### Environment where bug has been observed

- [ ] python 3.8
- [ ] python 3.9
- [ ] python 3.10
- [X] python 3.11
- [ ] python 3.12
- [X] macosx
- [ ] rhel7
- [ ] rhel8
- [X] local queue
- [ ] lsf queue
- [ ] slurm queue
- [ ] openPBS queue
- [ ] used flag `--enable-scheduler`

### PR DESCRIPTION
**Pull request by jonathan-eq**
**Issue**
Resolves #8691

**Approach**
_Short description of the approach_

(Screenshot of new behavior in GUI if applicable)

image

When specifying failed realizations:
image

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure unit tests pass locally after every commit (`git rebase -i main
      --exec 'pytest tests/ert/unit_tests -n logical -m "not integration_test"'`)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: src/ert/validation/realizations_in_ensemble_argument.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+from .rangestring import rangestring_to_list
+from .validation_status import ValidationStatus
+
+if TYPE_CHECKING:
+    from ert.storage import Ensemble
+
+
+class RealizationsInEnsembleArgument(RangeStringArgument):
+    UNINITIALIZED_REALIZATIONS_SPECIFIED = (
+        "The specified realization(s) %s are not found in selected ensemble."
```
Comment (xjules) [inline src/ert/validation/realizations_in_ensemble_argument.py:13]: Can we append the `ensemble.name`?; ie.  "The specified realization(s) %s are not found in selected ensemble %s."?

Comment (xjules) [reply src/ert/validation/realizations_in_ensemble_argument.py:13]: Ignore :)

File: src/ert/validation/realizations_in_ensemble_argument.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+from .range_string_argument import RangeStringArgument
+from .rangestring import rangestring_to_list
+from .validation_status import ValidationStatus
+
+if TYPE_CHECKING:
+    from ert.storage import Ensemble
+
+
+class RealizationsInEnsembleArgument(RangeStringArgument):
```
Comment (xjules) [inline src/ert/validation/realizations_in_ensemble_argument.py:11]: `EnsembleRealizationsArgument`?

File: src/ert/validation/realizations_in_ensemble_argument.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            validation_status.addToMessage(
+                RealizationsInEnsembleArgument.UNINITIALIZED_REALIZATIONS_SPECIFIED
+                % str(invalid_realizations)
+            )
+            return validation_status
+
+        validation_status.setValue(token)
+        return validation_status
+
+    def _validate_selected_realization_exist(self, realization: int) -> bool:
```
Comment (xjules) [inline src/ert/validation/realizations_in_ensemble_argument.py:50]: Don't understand this one. Why we need to have responses when we are about to run evaluate experiment?

Comment (jonathan-eq) [reply src/ert/validation/realizations_in_ensemble_argument.py]: ^I meant to do parameters, not responses :)

File: src/ert/validation/ensemble_realizations_argument.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        if not token:
+            return ValidationStatus()
+
+        validation_status = super().validate(token)
+        if not validation_status:
+            return validation_status
+        attempted_realizations = rangestring_to_list(token)
+
+        invalid_realizations = []
+        initialized_realization_ids = self.__ensemble.is_initalized()
```
Comment (xjules) [inline src/ert/validation/ensemble_realizations_argument.py:35]: now it makes sense :)

**Comment by xjules** [comment]
Can you squash the commits?

**Review by xjules — APPROVED** [review]
Nice PR @jonathan-eq ! :rocket:

**Pull request merged**